.github/workflows/beta-extensions.yml

```diff
@@ -70,7 +70,8 @@ jobs:
       - pre_run
     env:
       MINIFY_PRODUCTION_BUILD: true
-    # only run on commits with message 'Version release'
+    # only run on commits with message 'Version release' because this will often fail on PRs 
+    # due to reasons out of our control in the Firefox submission process
     if: github.event.pull_request.head.repo.full_name == github.repository && github.event.head_commit.message == 'Version release'
```

Comment:
i know this is not actually in scope of this PR, but I think I got this logic incorrect as it still doesn't run on the version pr. Any ideas what might work better?

Reply:
@aulneau  I think I see why this isn't working. When any workflow runs on a PR, it doesn't actually run from the latest commit of the HEAD branch (the branch being merged), instead it runs on a special git ref called `merge`. For [this PR](https://github.com/blockstack/stacks-wallet-web/pull/1466), the merge ref is `refs/remotes/pull/1466/merge`.

Merge refs behave like the BASE branch (the branch being merged _into_) post-merge. So a merge ref doesn't have the same commit message of the latest commit in the HEAD branch.

For the PR linked, the merge commit message is `Merge 6a2062ac9007d3fa7682e51ca522757d8ac3eac8 into 0532be1109aa8e0a99162bb6d109dae30fb69495`. You can see this message if you run:
```
git fetch origin +e88a3421a376d53dff49b4c2beb9a95c12793a19:refs/remotes/pull/1466/merge
git checkout --progress --force refs/remotes/pull/1466/merge
git log
```

A simple solution would be to search for `Merge` in the head_commit. Otherwise I think using `github.event.commits[0].message` will grab the commit you want from the PR. I'll change it to that to be tried on the next release.